samigo/samigo-app/src/webapp/js/extendedTime.js

```diff
@@ -112,18 +137,27 @@ function copyListValuesForExtTime(itemNum) {
 
 }
 
+function formatOrNull(date) {
+    if (date != null) {
+	return moment(date).format('MM/DD/YYYY HH:mm');
```

Comment:
Yes. What is stored in the database is typical US English date time format. However the picker works in whatever language is chosen.

The Javascript takes the ISO format and turns it into US English to store in the database.

The date picker is initialized from that format and should then localize it.

I did try it in a different Locale, but I need to do so again.

On May 10, 2016, at 2:36 PM, Sam Ottenhoff <notifications@github.com<mailto:notifications@github.com>> wrote:


In samigo/samigo-app/src/webapp/js/extendedTime.js<https://github.com/sakaiproject/sakai/pull/2495#discussion_r62727449>:

> @@ -112,18 +137,27 @@ function copyListValuesForExtTime(itemNum) {
>
>  }
>
> +function formatOrNull(date) {
> +    if (date != null) {
> +     return moment(date).format('MM/DD/YYYY HH:mm');


Does this work for Europeans?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/sakaiproject/sakai/pull/2495/files/89088cf609705beaca52a175f2f49f4e0c303e32#r62727449>